### ISSUE TITLE
Always 

### ISSUE THREAD

**Issue by swekkiekekkie** [gitbutlerapp/gitbutler#4209]
### Version

BOTH 0.12.7, Nightly Version: 0.5.568 

### Operating System

Linux

### Distribution Method

deb (Linux)

### Describe the issue

From installing and first opening an existing git repo, all lanes are in constant file dragging state and therefore everything is covered in "move here" or "squash commit", making git butler completely unusable.

**Comment by krlvi** [comment]
Hey there! thanks for reporting. Which Linux distribution (and version) are you using? We have seen this issue before with the app image distribution, but never with the .dev one, and I wonder if this could be related to a specific version

**Comment by krlvi** [comment]
@ndom91 I think this is the same one that you were investigating last week

**Comment by ndom91** [comment]
Yeah this seems to be the same issue as I noticed on the AppImage releases. I'm looking more itno this today. Will assign this issue to myself :+1: 

@swekkiekekkie we'll keep you posted!

**Comment by swekkiekekkie** [comment]
Thank you sirs for your efforts and AMAZING response time!!! keep it up

**Comment by tzual** [comment]
reproduced with appimage using fedora 40, gnome 46, x11, linux kernel 6.9.5

really looking forward to start using Gitbutler on my workstation!

**Comment by ndom91** [comment]
@tzual you're using gnome 46 with x11? Not wayland? 

@swekkiekekkie can you confirm if your Ubuntu installation is using x11 or wayland?

Seems like both x11 and wayland and appimage and deb packaging formats are experiencing this. Meaning it looks to be a general Linux / `libwebkit2gtk-4.x` issue :thinking:

**Comment by tzual** [comment]
> @tzual you're using gnome 46 with x11? Not wayland? 
> 
> @swekkiekekkie can you confirm if your Ubuntu installation is using x11 or wayland?
> 
> Seems like both x11 and wayland and appimage and deb packaging formats are experiencing this. Meaning it looks to be a general Linux / `libwebkit2gtk-4.x` issue :thinking: 

yeah, i had some issues with wayland, so I had to switch back to x11 and never switched back, but the upside is i get to check stuff like this in a less conventional setup 😀

i think this gives compelling evidence that the problem affects multiple distributions and bundles, and its source is probably around what you suggested 👌

**Comment by stedwick** [comment]
Unfortunately, this happened to me when I upgraded my AppImage yesterday. Now GitButler is currently unusable.

![Screenshot from 2024-07-02 07-35-42](https://github.com/gitbutlerapp/gitbutler/assets/63236/1ca8a1ca-b8b7-4718-9eb5-75bc2f310de2)

**Comment by ndom91** [comment]
We're actively looking into it, but for now we'd recommend Linux users stick to these releases:

- Nightlies at `0.5.552` or earlier
- Production releases `0.12.5` or earlier

**Comment by hr1ycfl** [comment]
You can provide the download address of version 0.12.5 for intel mac. On intel mac 0.12.7, move here will always be displayed.

**Comment by swekkiekekkie** [comment]
Just installed Nightly Version: 0.5.575 and no longer have this issue.

**Comment by krlvi** [comment]
> Just installed Nightly Version: 0.5.575 and no longer have this issue.

this is good to hear! I am wondering, is anybody else also able to confirm that the latest nigthtly fixes the issue? (can be grabbed from over here https://app.gitbutler.com/downloads)

**Comment by IguJl15** [comment]
Had the same problem and **found a work around**. Hopefully, this can assist someone else.

For me, the problem started on 0.12.5 -> 0.12.6 update and still on latest 0.12.7 update.

#### What worked FOR ME:

I'm using GNOME 46.1 and managed to "solve" the problem by downloading the Flatpak version from the "Software" app (GNOME software store). Unfortunately, other issues have arisen with this, but Git Butler was functional (Flatpak problems are not relevant here in this conversation).

---

> We're actively looking into it, but for now we'd recommend Linux users stick to these releases:
> 
> * Nightlies 0.5.552 or earlier ([AppImage](https://releases.gitbutler.com/releases/nightly/0.5.552-1008/linux/x86_64/git-butler-nightly_0.5.552_amd64.AppImage.tar.gz) | [deb](https://releases.gitbutler.com/releases/nightly/0.5.552-1008/linux/x86_64/git-butler-nightly_0.5.552_amd64.deb))
> * Production releases 0.12.5 or earlier ([AppImage](https://releases.gitbutler.com/releases/release/0.12.5-1004/linux/x86_64/git-butler_0.12.5_amd64.AppImage.tar.gz) | [deb](https://releases.gitbutler.com/releases/release/0.12.5-1004/linux/x86_64/git-butler_0.12.5_amd64.deb))


I will test the Nightly version and return tomorrow to provide feedback.

---

<details><summary>My workstation</summary>
<p>

```
NAME="openSUSE Tumbleweed"
\# VERSION="20240629"
ID="opensuse-tumbleweed"
ID_LIKE="opensuse suse"
VERSION_ID="20240629"
PRETTY_NAME="openSUSE Tumbleweed"

DE: GNOME 46.2 
WM: Mutter

Linux 6.9.6-1-default
```

</p>
</details>

**Comment by isgj** [comment]
Maybe another related issue, the toggles are not clickable (I can't see the state)

![image](https://github.com/gitbutlerapp/gitbutler/assets/20517048/1c6d2297-5b9b-4b26-9e62-d5c31aba02f1)

**Comment by ndom91** [comment]
Any flatpak release is an unofficial community package, probably on an earlier release. 

12.5 was the latest production release without this issue on Linux, regardless of the packaging format

**Comment by ndom91** [comment]
Please give `0.5.586` (latest nightly) a shot, it's working for me again :pray:

**Comment by isgj** [comment]
Is there an AppImage build for nightly? Deb packages do not install in Ubuntu 24.04 because of a missing dependency

![image](https://github.com/gitbutlerapp/gitbutler/assets/20517048/86db4d16-6bfb-4b7d-b160-1fc9e05fa66e)

24.04 has another version

![image](https://github.com/gitbutlerapp/gitbutler/assets/20517048/55fac306-f60c-4105-b0c0-0d5fdc158b8e)

**Comment by ndom91** [comment]
> Is there an AppImage build for nightly? Deb packages do not install in Ubuntu 24.04 because of a missing dependency
> 
> ![image](https://github.com/gitbutlerapp/gitbutler/assets/20517048/86db4d16-6bfb-4b7d-b160-1fc9e05fa66e)
> 
> 24.04 has another version
> 
> ![image](https://github.com/gitbutlerapp/gitbutler/assets/20517048/55fac306-f60c-4105-b0c0-0d5fdc158b8e)
> 

Yeah sorry the Linux for the appimage isn't published atm, here it is for `0.5.586`:

https://releases.gitbutler.com/releases/nightly/0.5.586-1052/linux/x86_64/git-butler-nightly_0.5.586_amd64.AppImage.tar.gz

**Comment by ndom91** [comment]
Okay one more try :joy:. Can you give `589` a shot?

- AppImage: https://releases.gitbutler.com/releases/nightly/0.5.589-1055/linux/x86_64/git-butler-- nightly_0.5.589_amd64.AppImage.tar.gz
- deb: https://releases.gitbutler.com/releases/nightly/0.5.589-1055/linux/x86_64/git-butler-nightly_0.5.589_amd64.deb

**Comment by IguJl15** [comment]
**Nightly version solved the problem for me!**

> Any flatpak release is an unofficial community package, probably based on an earlier release. For Linux we only publish rpm's, .deb's and AppImage at the moment.

So I'm taking the Nightly RPM package

---

Nightly version working for me: `.5.580`, `.5.589`
*I haven't tried in between versions*

Hope it works for @isgj too :pray:

**Comment by isgj** [comment]
the overlay is fixed, but the toggles not (not only in this screen, everywhere)

![image](https://github.com/gitbutlerapp/gitbutler/assets/20517048/85a29ed2-7913-4eef-bd33-e18c67deb633)

**Comment by ndom91** [comment]
@isgj which version were you using there?

**Comment by isgj** [comment]
I think it was `0.5.592`, but it's fixed in the later releases
Thanks
